Makefile

```
@@ -1,3 +1,5 @@
+CLUSTER_NAME := cruisekube-cluster
+
 .PHONY: help test serve-docs build-docs fetch-contributors
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)